src/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -132,11 +89,34 @@ public static Process ConfigureProcess(string docUrl)
                     Arguments = docUrl
                 };
             }
-            
+
             return new Process
             {
                 StartInfo = psInfo
             };
         }
+
+        public int Execute()
+        {
+            if (BuiltInCommandsCatalog.Commands.TryGetValue(
+                _appliedOption.Arguments.Single(),
+                out BuiltInCommandMetadata builtIn))
+            {
+                var process = ConfigureProcess(builtIn.DocLink);
+                process.Start();
+                process.WaitForExit();
+            }
+            else
+            {
+                Reporter.Error.WriteLine(
+                    string.Format(
+                        LocalizableStrings.CommandDoesNotExist,
+                        _appliedOption.Arguments.Single()));
+                Reporter.Output.WriteLine(HelpUsageText.UsageText);
+                return 1;
+            }
+            return 0;
```

Comment:
It would probably read a little more clearly if you move this return statement into the initial if block.